### PR TITLE
add screenshot

### PR DESCRIPTION
**Pull request by james2doyle**

### CODE REVIEW

**Comment by james2doyle** [comment]
@Drarok I don’t have dark mode but if you take one and upload it here I will add it to my PR. Can you help me?

**Comment by Drarok** [comment]
Sure thing, how's this? I tried to show the same options as your one:
Screenshot 2019-04-02 at 11 08 04

**Comment by james2doyle** [comment]
@Drarok looks good to me. I will add it in

File: README.markdown [hunk 1]
```diff
@@ -15,3 +15,9 @@ GitX**](https://github.com/gitx/gitx.github.io/issues/1).
 
 See [the wiki page](https://github.com/gitx/gitx/wiki/Build-instructions)
 for build instructions.
+
+### Sreenshots
```
Comment (Drarok) [inline README.markdown]: Sreenshots -> Screenshots?

Comment (james2doyle) [reply README.markdown]: Fixed!

**Review by jspiro — APPROVED** [review]

**Comment by hannesa2** [comment]
Aren't the pictures too big ? 
I like to scroll less. but we can change this after merge as well. I would cut the bottom empty place

image

**Comment by jspiro** [comment]
I thought the same but was lazy about new screenshots. You want to take some new ones?

**Review by hannesa2 — APPROVED** [review]
I will change after merge

**Comment by hannesa2** [comment]
I see, I've to finish CI first before to be able to merge

**Comment by jspiro** [comment]
It's merged!

**Pull request merged**